assignments/models.py

```diff
@@ -121,3 +121,30 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class Student(models.Model):
+    assignment = models.ForeignKey(Assignment, related_name='students')
+    school = models.CharField(max_length=256)
```

Comment:
Oh, then I'm sorry, I just thought that this was using MySQL as well. Well then I guess these values are ok since there are not the same kind of limitations on Postgres as on MySQL. These values are probably fine then.